_stbt/mask.py

```diff
@@ -97,10 +102,13 @@ def __eq__(self, o):
 
     def __hash__(self):
         if self._array is not None:
-            from _stbt.xxhash import Xxhash64
-            h = Xxhash64()
-            h.update(numpy.ascontiguousarray(self._array).data)
-            return hash((self._array.shape, h.hexdigest(), self._invert))
+            if Xxhash64:
+                h = Xxhash64()
+                h.update(numpy.ascontiguousarray(self._array).data)
```

Comment:
Potential future optimisation: avoid a copy by iterating over rows. This would need to be written in C.